txdbus/authentication.py

```diff
@@ -372,7 +380,7 @@ def _get_cookies(self,timefunc=time.time):
         cookies = list()
         f = None
         try:
-            f = open(self.cookie_file, 'r')
+            f = open(self.cookie_file, 'rb')
```

Comment:
BTW: You could probably use `with open(self.cookie_file, 'rb'):` here to make this a lot more readable